src/software/thunderscope/play/qlearninginfo_widget.py

```diff
@@ -0,0 +1,136 @@
+import time
+import logging
+import pathlib
+
+import pyqtgraph as pg
+import pyqtgraph.console as pg_console
+from proto.q_learning_pb2 import LinearQFunctionInfo
+from software.networking.unix.threaded_unix_listener import ThreadedUnixListener
+from software.thunderscope.constants import SAVED_Q_FUNCTION_WEIGHTS_PATH
+from pyqtgraph.Qt import QtCore, QtGui
+from pyqtgraph.Qt.QtWidgets import *
+from proto.import_all_protos import *
+from google.protobuf.json_format import MessageToDict
+from software.thunderscope.common.common_widgets import set_table_data
+
+from software.thunderscope.thread_safe_buffer import ThreadSafeBuffer
+
+
+class QLearningInfoWidget(QWidget):
+
+    # empirically makes even bolded items fit within columns
+    ITEM_SIZE_HINT_WIDTH_EXPANSION = 10
```

Comment:
by PEP257, docstrings should be below these attributes: https://peps.python.org/pep-0257/#what-is-a-docstring